src/main/java/org/broadinstitute/hellbender/Main.java

```diff
@@ -78,14 +84,19 @@ public Object instanceMain(final String[] args) {
     }
 
     /**
-     * Override this if you want to include different java packages to search for classes that extend CommandLineProgram. *
+     * The entry point to GATK from commandline.
+     * Note: this is the only method that is allowed to call System.exit (because gatk tools may be run from test harness etc)
      */
     public static void main(final String[] args) {
         try {
             final Object result = new Main().instanceMain(args, getPackageList(), "");
             if (result != null) {
               System.out.println("Tool returned:\n" + result);
             }
+        } catch (final UserException.CommandLineException e){
```

Comment:
I think that stack traces could be helpful for developers for command line exceptions.  I see at least a few places in the code where they're thrown outside of the command line parser, and it would be tricky to track them down without the stack trace.  

It's also a bit weird to have the error be formatted differently (i.e. without the *****s).